### ISSUE TITLE
NuGet based Logic App (Standard) does not build in Visual Studio

### ISSUE THREAD

**Issue by gastonmuijtjens** [Azure/logicapps#323]
I created a new single-tenant _Logic App (Standard)_ using the Logic Apps (Standard) extension in Visual Studio Code. I followed the [documentation](https://docs.microsoft.com/en-us/azure/logic-apps/create-single-tenant-workflows-visual-studio-code#enable-built-in-connector-authoring) to migrate to a NuGet package-based (.NET) project.

When I try to build the project with the .NET CLI: `dotnet build -c Debug` and `dotnet build -c Release`, it builds fine.

However, when I try to build the project within Visual Studio 2019 with the `Debug` configuration, I'm getting the following error:
```
The "DebugSymbolGenerator" task failed unexpectedly.
System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
File name: 'System.Runtime, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at Microsoft.Azure.Workflows.BuildTasks.DebugSymbolGenerator.DebugSymbolGenerator.LogMessage(String message)
   at Microsoft.Azure.Workflows.BuildTasks.DebugSymbolGenerator.DebugSymbolGenerator.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
```

When I try the same with the `Release` configuration, the project builds just fine.
I attached an example solution as a `.zip` file that generates the above error:

[TestLogicApp.zip](https://github.com/Azure/logicapps/files/6765996/TestLogicApp.zip)

- Visual Studio 2019 16.10.3
- Microsoft.Azure.Workflows.WebJobs.Extension 1.1.3
- Microsoft.NET.Sdk.Functions 3.0.9

**Comment by hongzli** [comment]
Hi @gastonmuijtjens , thanks for reporting this issue. The NuGet-based project was designed to work with Visual Studio Code and dotnet/functions CLI but not Visual Studio. 

Do you have a scenario that specifically requires using Visual Studio?

**Comment by kaylumah** [comment]
@hongzli we initially tried the NuGet-based project since our LogicApp was the ONLY project in our solution that was not part of the solution. It would allow for managing the entire project from single IDE (ie VS2019).

I could understand not being able to run it via VS2019, but not being to build a CSProj independent from IDE or tool seems like an oversight to me.

As far as I know (correct me if I am wrong here) LogicApps use the functions runtime, and C# csproj-style projects can both be run and build from VisualStudio.

Could you clarify why it would only work in VSCode?

**Comment by gastonmuijtjens** [comment]
@hongzli I would like to add that since our Logic App is part of a solution (.sln) together with other related non-Logic App projects we currently have to unload the Logic App project within Visual Studio in order to build. Also the Visual Studio build task within a Windows-agent based build pipeline in Azure DevOps would not work and requires a separate build with the dotnet cli instead.

**Comment by gastonmuijtjens** [comment]
@hongzli Any update on this?

**Comment by hongzli** [comment]
Unfortunately we aren't currently supporting VS development, but it is on the horizon and we will be adding support for it.

**Comment by kaylumah** [comment]
@hongzli do you have any timeline you can share on this? We know need to rely on a sln filter to build or project (excluding the logicapp) which is not ideal

**Comment by hongzli** [comment]
We still don't have a timeline for official VS support that we can share, but as a lot of users are requesting this, we will be shipping a fix for this early next year (Jan or Feb) to unblock building in VS

**Comment by kaylumah** [comment]
Hi @hongzli since its Feb 1st I am wondering is if you can share an update planning on this issue?
